src/Microsoft.OData.Core/ODataMessageWriterSettings.cs

```diff
@@ -114,7 +114,7 @@ public ODataMessageWriterSettings(ODataVersion? version)
         }
 
         /// <summary>
-        /// Gets or sets library compatibility version. Default value is <see cref="Microsoft.OData.ODataLibraryCompatibility.Latest"/>,
+        /// Gets or sets library compatibility version. Default value is <see cref="Microsoft.OData.ODataLibraryCompatibility.None"/>,

```

Comment:
8.x change?

Reply:
There's no 'Latest' enum member for ODataLibraryCompatibility, And I noticed that in the constructor, 'None' is used for default value. I didn't see any changes within 8.x version for this part..

Reply:
Okay; found Elizabeth's [commit ](https://github.com/OData/odata.net/commit/fde7d8e0f02879799ac06a4de3dddc2613d054af) that changed this. Thanks for catching the comment.